pkg/exporter/convert_flp_test.go

```diff
@@ -0,0 +1,376 @@
+package exporter
+
+import (
+	"net"
+	"testing"
+	"time"
+
+	"github.com/netobserv/flowlogs-pipeline/pkg/config"
+	"github.com/netobserv/flowlogs-pipeline/pkg/utils"
+	"github.com/netobserv/netobserv-ebpf-agent/pkg/decode"
+	"github.com/netobserv/netobserv-ebpf-agent/pkg/ebpf"
+	"github.com/netobserv/netobserv-ebpf-agent/pkg/flow"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/proto"
+)
+
+func TestConvertFLP(t *testing.T) {
+	decoder := decode.Protobuf{}
+
+	someTime := time.Now()
+	var someDuration time.Duration = 10000000 // 10ms
+
+	tests := []struct {
```

Comment:
@msherif1234 FYI the tests here are the same that were previously in `decode_protobuf_test` except that they're now used to test both pb conversion and direct FLP conversion, making sure they both produce the same result

Reply:
this test coverting from ebpf record to flp configmap but it seems u deleted from gRPC to configmap in case this mode isn't set ?

Reply:
cf my next comment below